content/ar/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -0,0 +1,114 @@
+            </ul>
+        </div>
+    </div>
+
+    <br>
+
+    <div class="row">
+        <div class="col-md-9">
+            <h2>ماذا يمكن لكوبيرنيتيس أن يفعل لك؟</h2>
+            <p>مع خدمات الويب الحديثة، يتوقع المستخدمون توافر التطبيقات على مدار الساعة طوال الأسبوع، ويتوقع المطورون نشر نسخ جديدة من هذه التطبيقات عدة مرات في اليوم. تساعد تقنية التعبئة في تحقيق هذه الأهداف، حيث تمكن من تعبئة البرمجيات لتحقيق هذه الأهداف، مما يتيح إطلاق التطبيقات وتحديثها دون توقف. يساعدك كوبيرنيتيس في التأكد من أن هذه التطبيقات المعبأة في حاويات تعمل حيث ومتى تريد، ويساعدها في العثور على الموارد والأدوات التي تحتاج إليها للعمل. كوبيرنيتيس هو منصة مفتوحة المصدر جاهزة للإنتاج، تم تصميمها بخبرة جوجل المتراكمة في تنظيم الحاويات، مع أفضل الأفكار من المجتمع.</p>
+        </div>
+    </div>
+
+    <br>
+
+    <div id="basics-modules" class="content__modules">
+      <h2>أساسيات كوبيرنيتيس</h2>
+      <div class="row">
+        <div class="col-md-12">
+          <div class="row">
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/ar/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_01.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/ar/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><h5>1. إنشاء عنقود كوبيرنيتيس</h5></a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/ar/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_02.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/ar/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/"><h5>2. نشر التطبيق</h5></a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/ar/docs/tutorials/kubernetes-basics/explore/explore-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_03.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/ar/docs/tutorials/kubernetes-basics/explore/explore-intro/"><h5>3. استكشاف التطبيق</h5></a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-12">
+          <div class="row">
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/ar/docs/tutorials/kubernetes-basics/expose/expose-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_04.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/ar/docs/tutorials/kubernetes-basics/expose/expose-intro/"><h5>4. تعريض التطبيق للعموم</h5></a>
```

Comment:
أرجو النظر في ترجمة expose هناك عدة خيارات.
```suggestion
                  <a href="/ar/docs/tutorials/kubernetes-basics/expose/expose-intro/"><h5>4. إبراز التطبيق خارج العنقود</h5></a>
```